features/full_tests/batch_1/identifying_crashes_on_launch.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Identifying crashes on launch
+
+    Scenario: Escaping from a crash loop by reading LastRunInfo in a JVM error
+        When I run "JvmCrashLoopScenario" and relaunch the app
+        And I configure Bugsnag for "JvmCrashLoopScenario"
+        When I run "JvmCrashLoopScenario"
+        And I wait to receive 2 errors
+        Then the error is valid for the error reporting API version "4.0" for the "Android Bugsnag Notifier" notifier
+        And the exception "message" equals "First crash"
+        And the event "metaData.LastRunInfo.crashed" is null
+        And the event "metaData.LastRunInfo.crashedDuringLaunch" is null
+        And the event "metaData.LastRunInfo.consecutiveLaunchCrashes" is null
+        And I discard the oldest error
+        Then the error is valid for the error reporting API version "4.0" for the "Android Bugsnag Notifier" notifier
+        And the exception "message" equals "Safe mode enabled"
+        And the event "metaData.LastRunInfo.crashed" is true
+        And the event "metaData.LastRunInfo.crashedDuringLaunch" is true
+        And the event "metaData.LastRunInfo.consecutiveLaunchCrashes" equals 1
+
+#    Awaiting implementation of PLAT-5978
+#    Scenario: Escaping from a crash loop by reading LastRunInfo in an NDK error
```

Comment:
When the time comes to enable this, I think you could just add some steps to the above scenario, rather than creating a new one.  I.e. JVM crash -> NDK crash -> Safe mode.  This would save a bit of test run time and also verify that consecutiveLaunchCrashes is incremented beyond 1.

Reply:
Added a note to do this onto the ticket, so that this can get merged down quickly.